testkit/scaladsl/src/main/scala/com/lightbend/lagom/scaladsl/testkit/ServiceTest.scala

```diff
@@ -344,6 +344,8 @@ object ServiceTest {
     }
 
     if (setup.cassandra || setup.jdbc) {
+      // We should eagerly init SlickProvider before test a JDBC persistence
+      if (setup.jdbc) Try(lagomApplication.asInstanceOf[SlickProviderComponents]).map(_.slickProvider)
```

Comment:
what is the relation of this change with upgrading to Play v2.8.2?

Could we make it in another PR so we can track things better?

Reply:
Hmm, I see that this is a backport and it was on the original PR.

Reply:
Even if this is a backport, I had second thoughts (https://github.com/lagom/lagom/pull/2934#discussion_r451628341) wrt the impact this change has.

Now that's merged the question is: are we comfortable with the jdbc transitive dependency even for cassandra users? If not, we should roll this back and implement a different strategy. If yes, then we can move on to a 1.6.3 release.